tests/records/test_api.py

```diff
@@ -133,6 +133,26 @@ def test_draft_parent_state_hard_delete(app, db, location):
     assert RecordMetadata.query.count() == 0
 
 
+def test_new_draft_of_published_record_doesnt_override_next_draft_id(app, db, location):
+    # Fake a published record without a draft. That's the situation we get after
+    # cleanup_drafts has run on a newly published record
```

Comment:
Can you add in the test the "cleanup" action as well so we are sure about the fix?

Reply:
done